ga4gh/datamodel/variants.py

```diff
@@ -47,6 +48,17 @@ def convertVCFGenotype(vcfGenotype, vcfPhaseset):
     return genotype, phaseset
 
 
+class CompoundVariantId(datamodel.CompoundId):
+    """
+    The compound id for a variant
+    """
+    fields = ['datasetId', 'vsId', 'chrom', 'pos', 'md5']
```

Comment:
but the variantSetId is actually a combination of these fields (see `variantSetId` in the `comboFields` attr)